tests/SixLabors.Fonts.Tests/Tables/AdvancedTypographic/Gsub/GSubTableTests.cs

```diff
@@ -60,8 +26,8 @@ public void RenderArabicCharacters_WithIsolatedForm_Works()
             });
 
             // assert
-            Assert.Equal(expectedControlPoints.Length, rendererTtf.ControlPoints.Count);
-            Assert.True(rendererTtf.ControlPoints.SequenceEqual(expectedControlPoints));
+            Assert.Single(rendererTtf.GlyphKeys);
+            Assert.Equal(expectedGlyphIndex, rendererTtf.GlyphKeys[0].GlyphIndex);
```

Comment:
```suggestion
            var glyphKey = Assert.Single(rendererTtf.GlyphKeys);
            Assert.Equal(expectedGlyphIndex, glyphKey.GlyphIndex);
```

makes it slightly easier to read in my opinion

Reply:
yes, I agree